packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -3911,6 +3922,7 @@ function commitGestureOnRoot(
     pendingTransitionTypes,
     flushGestureMutations,
     flushGestureAnimations,
+    reportViewTransitionError,
```

Comment:
Does it make sense to run this in the context of `finishedWork` or is that not a relevant Owner of a potential error?

Reply:
`finishedWork` is just the `HostRoot` so it has no owner at all.

In general these errors has no owner at all. There are other warnings we can add earlier to prevent some of these errors from happening in the first place. For example, we should have a warning when two ViewTransition components are mounted with the same name. Even if they don't happen to be part of the current transition and those could have owners.